packages/zowe-explorer/src/trees/uss/UssFSProvider.ts

```diff
@@ -649,11 +660,7 @@ export class UssFSProvider extends BaseProvider implements vscode.FileSystemProv
                 }
             }
         } else {
-            const fileEntry = this.lookup(source, true);
-            if (fileEntry == null) {
-                return;
-            }
-
+            const fileEntry = this.lookup(source);
```

Comment:
Dumb question: 
Isn't this a breaking change in behavior?

My theory:
Having this function (copyTree) throw an error may ultimately result in us stopping a `pasteUssTree` operatation. Compared to returning right away, we might've allowed for a partial upload. However, I believe it's probably best to fail right away instead of having to keep track of which URIs failed to then report that in an error notification (which I don't think we were doing anyways)

Reply:
When copying from a `source` URI to `destination` URI, we have to make sure that the source URI exists in the file system. If the lookup fails within this branch, it's because the URI is pointing to a single file - and since that file no longer exists in the file system, and we cannot copy it as a result. That said, we will still do a partial upload for any part of the tree that's being uploaded, since this function is recursive, and it will simply fail at that file. Once the failure occurs, we'll stop copying the remainder of the tree, so the behavior should be the same.

Reply:
Hey, 
Thanks for taking the time to explain offline.
Partial upload will still occur and based on how the recursion iterates through folders, it will be clear to end users where we stopped uploading. 🙏🏽 
Again, thanks for clarifying 🥳